ompi/mca/mtl/psm2/mtl_psm2.c

```diff
@@ -171,6 +174,15 @@ int ompi_mtl_psm2_module_init(int local_rank, int num_local_procs) {
     /* register the psm2 progress function */
     opal_progress_register(ompi_mtl_psm2_progress);
 
+#if OPAL_CUDA_SUPPORT
+    ompi_mtl_psm2.super.mtl_flags |= MCA_MTL_BASE_FLAG_CUDA_INIT_DISABLE;
+
+    cuda_env = getenv("PSM2_CUDA");
+    if (!cuda_env || ( strcmp(cuda_env, "0") == 0) )
```

Comment:
Is `PSM2_CUDA` an environment variable that is exported by the PSM2 library?  Or is that an env variable that is solely being used by Open MPI?  If it is solely used by the PSM2 support in Open MPI, it should be an MCA variable.

Reply:
It is an envvar exported by PSM2 library.

Reply:
it is actually not "exported", but "read" by PSM2.